include/nuttx/sched.h

```diff
@@ -480,6 +533,9 @@ struct task_group_s
 
   /* Thread local storage ***************************************************/
 
+#ifndef CONFIG_MM_KERNEL_HEAP
+  struct task_info_s tg_info_;
```

Comment:
CONFIG_TLS_ALIGNED should be prefer than CONFIG_MM_KERNEL_HEAP

Reply:
why? task_info is different with tls

Reply:
Yes, you are right. I confuse between tls_info and task_info.